src/app/parcels/ParcelsPage.tsx

```diff
@@ -300,10 +366,14 @@ const ParcelsPage: React.FC<{}> = () => {
                         setRange={setPackingDateRange}
                     ></DateRangeInputs>
                 </ControlContainer>
-                <ActionBar data={tableData} selected={selected} />
+                <ActionBar
+                    selectedParcels={selectedRowIndices.map((index) => tableData[index])}
+                    onDeleteParcels={deleteParcels}
+                    setIsLoading={setIsLoading}
+                />
             </PreTableControls>
             {isLoading ? (
-                <></>
+                <Loading />
```

Comment:
The parcels page is already wrapped in `<main>` and Loading has `<main>` too so we would be showing
```
<main>
  ...
  <main>
    ...
  </main>
  ...
</main>
```

I think we should just use
```
<Centerer>
  <CircularProgress />
</Centerer>
```
or 
```
<CircularProgress />
```

Reply:
Have done (with centerer)